classes/PodsUI.php

```diff
@@ -4653,9 +4656,21 @@ public function pagination( $header = false ) {
 
 		if ( false !== $this->pagination ) {
 			if ( 1 < $total_pages ) {
+				$first_link    = esc_url( $request_uri . ( $append ? '&' : '?' ) . 'pg' . $this->num . '=1' );
+				$prev_link     = esc_url( $request_uri . ( $append ? '&' : '?' ) . 'pg' . $this->num . '=' . max( $this->page - 1, 1 ) );
+				$next_link     = esc_url( $request_uri . ( $append ? '&' : '?' ) . 'pg' . $this->num . '=' . min( $this->page + 1, $total_pages ) );
+				$last_link     = esc_url( $request_uri . ( $append ? '&' : '?' ) . 'pg' . $this->num . '=' . $total_pages );
+
+				$classes = '';
+				if ( 1 < $this->page ) {
+					$classes .= ' disabled';
+				}
+				if ( is_admin() ) {
+					$classes .= ' button';
+				}
 				?>
-				<a class="first-page<?php echo esc_attr( ( 1 < $this->page ) ? '' : ' disabled' ); ?>" title="<?php esc_attr_e( 'Go to the first page', 'pods' ); ?>" href="<?php echo esc_url( $request_uri . ( $append ? '&' : '?' ) . 'pg' . $this->num . '=1' ); ?>">&laquo;</a>
-				<a class="prev-page<?php echo esc_attr( ( 1 < $this->page ) ? '' : ' disabled' ); ?>" title="<?php esc_attr_e( 'Go to the previous page', 'pods' ); ?>" href="<?php echo esc_url( $request_uri . ( $append ? '&' : '?' ) . 'pg' . $this->num . '=' . max( $this->page - 1, 1 ) ); ?>">&lsaquo;</a>
+				<a class="first-page<?php esc_attr_e( $classes ); ?>" title="<?php esc_attr_e( 'Go to the first page', 'pods' ); ?>" href="<?php echo $first_link; ?>">&laquo;</a>
```

Comment:
* :no_entry_sign: Missing $domain arg.
* :no_entry_sign: The $text arg must be a single string literal, not "$classes".
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$first_link'.

<sub>hash: 9d402a20fbb051bb57d0d321c8b8a335</sub>